firmware/src/core/Feeds.h

```diff
@@ -0,0 +1,25 @@
+#ifndef FEEDS_H
+#define FEEDS_H
+
+#include <Arduino.h>
+#include "Feed.h"
+#include "..\..\feeds\FeedFactory.h"
+
+struct Feeds {
+private:
+  Feed* _Feeds[MAXFEEDID];
+public:
+  Feed& registerFeed(const FeedId feedId) {
+    if (!_Feeds[feedId])
+    {
+        _Feeds[feedId] = FeedFactory::CreateFeed(feedId);
+    }
+  }
+
+  Feed& getFeed(const FeedId feedId) {
+    assert(_Feeds[feedId]);
```

Comment:
This suggests that all feeds are always created, I would advice against that.
Anything that is not needed, does not have to be loaded.